app/jobs/malware_scanning_job.rb

```diff
@@ -2,9 +2,10 @@
 
 class MalwareScanningJob < ApplicationJob
   def perform
-    Upload
-      .where(malware_scan_result: "pending")
-      .where("created_at < ?", 5.minutes.ago)
-      .find_each { |upload| MalwareScanJob.perform_later(upload_id: upload.id) }
+    uploads = Upload.where(malware_scan_result: "pending").where("created_at < ?", 5.minutes.ago)
+
+    return unless uploads.any?
+
```

Comment:
is this needed?

Reply:
the find_each seems to result in that error if there are no uploads present (which there aren't in staging)